### PR TITLE
fix(tdengine): remove the redundant table name in SQL template

### PR DESCRIPTION
**Pull request by lafirest**
remove the redundant table name in SQL template

Fixes [EMQX-9226 ](https://emqx.atlassian.net/browse/EMQX-9226)

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/{ce,ee}/(feat|perf|fix)-<PR-id>.en.md` and `.zh.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] If there should be document changes, a PR to emqx-docs.git is sent, or a jira ticket is created to follow up
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl [hunk 1]
```diff
@@ -49,6 +51,8 @@ add_default_username(Fields) ->
         fun
             ({username, OrigUsernameFn}) ->
                 {username, add_default_fn(OrigUsernameFn, <<"root">>)};
+            ({password, OrigUsernameFn}) ->
+                {password, add_default_fn(OrigUsernameFn, ?DEFAULT_PASSWORD)};
```
Comment (keynslug) [inline lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl:55]: Concept of _default password_ sounds somewhat strange to me, honestly. I think that if there's a passwordless auth, then connector should anticipate password being not defined. If there is no passwordless auth, then we should just mark the field required?

Comment (lafirest) [reply lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl:55]: Usually, The Tdengine use a default password, "data", users don't need to care about this password,  and the schema of `password` is not required, of course, we can change it to true, but in this way, users have to repeat the password which should have omitted.
So we should not force users to input this password again.

Comment (keynslug) [reply lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl:55]: I see, but I wouldn't really expect users to deploy TDengine DB with a default password in the wild.

Comment (thalesmg) [reply lib-ee/emqx_ee_connector/src/emqx_ee_connector_tdengine.erl:55]: https://github.com/emqx/emqx/pull/10269

File: changes/ee/fix-10201.en.md [hunk 2]
```diff
@@ -0,0 +1 @@
+In TDengine, Remove the redundant table name from the SQL template.
```
Comment (keynslug) [inline changes/ee/fix-10201.en.md:1]: Isn't that _database name_, according to https://docs.tdengine.com/taos-sql/table/#create-table ?

Comment (lafirest) [reply changes/ee/fix-10201.en.md:1]: Yes, it is `database` not `table`.

**Review by keynslug — APPROVED** [review]

File: changes/ee/fix-10201.en.md [hunk 3]
```diff
@@ -0,0 +1 @@
+In TDengine, Remove the redundant database name from the SQL template.
```
Comment (thalesmg) [inline changes/ee/fix-10201.en.md:1]:
```suggestion
In TDengine, removed the redundant database name from the SQL template.
```

**Review by JimMoen — APPROVED** [review]

**Review by HJianBo — APPROVED** [review]

**Pull request merged**